### ISSUE TITLE
[Design System Toolbox] update versions info to use the tag in the about screen

### ISSUE THREAD

**Issue by B3nz01d** [Orange-OpenSource/ouds-ios#1203]
### Description

**As a user** of the demo app
**I want** to see the about page
**So that** all the version numbers use the information tag component

as on the web documentation the tag should be using the following tag setup:
* the information status
* rounded
* muted
* text only
* in small size

### Checklist for Core Team

- [ ] Tokens have been updated (if needed)
- [ ] Documentations have been updated (if needed)
- [ ] Unit tests have been updated and still pass
- [ ] UI tests have been updated and still pass
- [ ] Retrocompatibility has been checked, or broken API have been flagged as deprecated with a workaround

**Comment by B3nz01d** [comment]
- [ ] Android: [# 979](https://github.com/Orange-OpenSource/ouds-android/issues/979)
- [ ] iOS: [# 1203](https://github.com/Orange-OpenSource/ouds-ios/issues/1203)
- [ ] Flutter: [# 501](https://github.com/Orange-OpenSource/ouds-flutter/issues/501)

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **1.0.0**
- Build number: **1765922908**
- Build tag: **8ba3d47**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New TestFlight BETA upload available** 🚀

Please, get the new build from *TestFlight* app, or contact the project maintainers (pylapp, ludovic35) to get access and test this issue implementation.

- Display name: **Design System Toolbox (BETA)**
- Version: **1.0.0**
- Build number: **1766009311**
- Build tag: **117a933**


_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by pylapp** [comment]
📣 **New release available** 🚀

Please, get the [new release from GitHub](https://github.com/Orange-OpenSource/ouds-ios-design-system-toolbox/releases/tag/1.0.0) and wait for App Store update to test it!

- Display name: **Design System Toolbox**
- Version: **1.0.0**
- Build number: **1766065729**



_🤖 This is an automated message sent with love thanks to *Fastlane* from our CI/CD pipeline 🤘_

**Comment by B3nz01d** [comment]
Finished during Sprint # 25
